public/app/features/alerting/unified/components/rules/RulesGroup.tsx

```diff
@@ -71,20 +82,34 @@ export const RulesGroup: FC<Props> = React.memo(({ group, namespace, expandAll }
     );
   } else if (rulesSource === GRAFANA_RULES_SOURCE_NAME) {
     if (folderUID) {
-      const baseUrl = `${config.appSubUrl}/dashboards/f/${folderUID}/${kbn.slugifyForUrl(namespace.name)}`;
+      const baseUrl = makeDashboardLink(folderUID);
```

Comment:
One more nit: I think it would be nice to open the folder page on the alerting tab